src/main/java/innovimax/mixthem/MixConstants.java

```diff
@@ -2,5 +2,9 @@
 
 public final class MixConstants {
```

Comment:
no common constant class
Constant should be grouped by topic in other classes

Reply:
ok